### PR TITLE
Add an option to select lines with areas of a multi-line chart

### PR DESCRIPTION
**Pull request by thomaschampagne**
**multilines.htm documentation screenshot:**

image

Use case: https://twitter.com/champagnethomas/status/972161893384228866

Linked to mozilla/metrics-graphics#252

### CODE REVIEW

**Comment by wlach** [comment]
@thomaschampagne sorry for the delay on this, this looks like a cool and useful feature to me -- @hamilton ?

File: src/js/common/data_graphic.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ MG.data_graphic = function(args) {
     show_rollover_text: true,
     show_confidence_band: null,                 // given [l, u] shows a confidence at each point from l to u
     xax_format: null,                           // xax_format is a function that formats the labels for the x axis.
-    area: true,
+    area: true,                                 // Can be also an array to select lines having areas (e.g. [1, 3])
```
Comment (hamilton) [inline src/js/common/data_graphic.js]: I'm wondering what the right API here is. We allow a user to pass in an array of arrays for data. Perhaps this should be an array of booleans. So if my `data` is three arrays, I should have to pass in `area: [false, false, true]`. I am not totally happy with how we are mostly passing these things in positionally, but from a semantic point of view I think this makes sense given how the library works.

Another possibility is that perhaps you can pass a function into array, which maps over each value of the data array (if you have an array of arrays) which can set `area: true` for that particular array.

@wlach what do you think about this from an API point of view?

Comment (thomaschampagne) [reply src/js/common/data_graphic.js]: I'll handle that in upcoming days.

**Comment by thomaschampagne** [comment]
@wlach @hamilton Work done !

**Comment by wlach** [comment]
This looks good except for the intellij files which were accidentally added, @thomaschampagne -- could you remove those?

**Comment by thomaschampagne** [comment]
Whoops :/ Sry for this. Not used to work with a .gitignore not matching my IDE.

I will do that for sure !

**Comment by thomaschampagne** [comment]
@wlach Files removed!

**Comment by wlach** [comment]
Thanks so much Thomas!

**Pull request merged**